# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: toolkit/actors/BrowserElementParent.jsm
# Commit: d7e962f7ca16
# Full Hash: d7e962f7ca165897bc5c1e32f8a1bcc72ab24629
# Author: Jonathan Watt <jwatt@jwatt.org>
# Date: 2020-01-22 21:48:55
# Description:
#   Bug 1607671. Back out changeset edf3df2adebb (bug 1602410) for regressions. kmag r=RyanVM
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D59785
# ==============================================================================

diff -r 61c5e64eb2b4 -r d7e962f7ca16 toolkit/actors/BrowserElementParent.jsm
--- a/toolkit/actors/BrowserElementParent.jsm	Wed Jan 22 14:41:47 2020 +0000
+++ b/toolkit/actors/BrowserElementParent.jsm	Wed Jan 22 14:43:46 2020 +0000
@@ -91,17 +91,6 @@
         }
         break;
       }
-      case "ContentRequestedPrint": {
-        // Note: this is not yet fully Fission compatible. That requires
-        // further work to change the PrintUtils API and the underlying code
-        // that it uses.
-        let bc = this.manager.browsingContext;
-        let topWin = bc.top.embedderElement.ownerGlobal;
-        if (topWin.PrintUtils) {
-          topWin.PrintUtils.printWindow(bc);
-        }
-        break;
-      }
     }
   }
 }
